privatek8s.tf

```
@@ -0,0 +1,122 @@
+data "azurerm_subscription" "jenkins" {
+}
+
+resource "azurerm_resource_group" "privatek8s" {
+  name     = "privatek8s"
+  location = var.location
+  tags     = local.default_tags
+}
+
+resource "random_pet" "suffix_privatek8s" {
+  # You want to taint this resource in order to get a new pet
+}
+
+resource "azurerm_subnet" "privatek8s_tier" {
+  name                 = "privatek8s-tier"
+  resource_group_name  = data.azurerm_resource_group.private_prod.name
+  virtual_network_name = data.azurerm_virtual_network.private_prod.name
+  address_prefixes     = ["10.242.0.0/16"]
+}
+
+#tfsec:ignore:azure-container-logging
```

```suggestion
#tfsec:ignore:azure-container-logging #tfsec:ignore:azure-container-limit-authorized-ips
```
Ignoring this tfsec error while finding a (good) way to set this IP range restriction (access via the VPN?)